ethdb/database.go

```diff
@@ -172,7 +174,6 @@ type Stater interface {
 type AncientStore interface {
 	AncientReader
 	AncientWriter
-	AncientStater
```

Comment:
Any particular reason for removing this?

Reply:
Because for ancientStore(freezer) itself, it doesn't know the **root ancient directory**, but only it's own path.
Only the eth.Database has the knowledge of root ancient.

Reply:
Ah, the outer db is the one resolving the subfolders, not the freezer itself. Ok, then I agree.